src/pages/staticPages/tools/mutationMapper/MutationMapperTool.tsx

```diff
@@ -587,4 +642,33 @@ export default class MutationMapperTool extends React.Component<
     protected handleLoadExampleGeneAndProteinChange() {
         this.inputText = require('raw-loader!./resources/standaloneMutationDataExampleWithGeneAndProteinChangeOnly.txt');
     }
+
+    @autobind
+    @action
+    protected handleReferanceGenomeSelectionChange(selection: string) {
+        if (selection === referenceGenome.GRCH37) {
+            this.referenceGenomeSelection = referenceGenome.GRCH37;
+            // if choose grch37, remove genomenexus_url_grch38 from localStorage
+            if (
+                getBrowserWindow().localStorage.getItem(
```

Comment:
rather that embedding the genome Id in the localStorage key, I think it would be better to have a single "referenceGenomeId" item that could represent any referenceGenome.